packages/node/src/login/oidc/ClientRegistrar.spec.ts

```diff
@@ -41,23 +48,239 @@ describe("ClientRegistrar", () => {
   }
 
   describe("getClient", () => {
-    it("isn't implemented", async () => {
+    it("Fails if there is not registration endpoint", async () => {
```

Comment:
Lowercase first char of `it("")` messages (it's supposed to read as a sentence, i.e. "this test, it **_fails if there is_* ..."):
```suggestion
    it("fails if there is not registration endpoint", async () => {
```

Reply:
Resolved in 723613e09a9a80001374bf3a8be9f5c37cb62ad1